pontoon/base/models.py

```diff
@@ -508,12 +514,16 @@ def adjust_stats(
         total_strings_diff,
         approved_strings_diff,
         fuzzy_strings_diff,
-        unreviewed_strings_diff
+        unreviewed_strings_diff,
+        for_dummy_project
     ):
         self.total_strings = F('total_strings') + total_strings_diff
-        self.approved_strings = F('approved_strings') + approved_strings_diff
-        self.fuzzy_strings = F('fuzzy_strings') + fuzzy_strings_diff
-        self.unreviewed_strings = F('unreviewed_strings') + unreviewed_strings_diff
+        self.approved_strings = (F('approved_strings') +
+                                 approved_strings_diff, 0)[for_dummy_project]
+        self.fuzzy_strings = (F('fuzzy_strings') +
+                              fuzzy_strings_diff, 0)[for_dummy_project]
+        self.unreviewed_strings = (F('unreviewed_strings') +
+                                   unreviewed_strings_diff, 0)[for_dummy_project]
```

Comment:
This is quite hard to read. I propose instead: 

```python
if for_dummy_project:
    # We do not count stats for dummy projects. 
    self.approved_strings = 0
    self.fuzzy_strings = 0
    self.unreviewed_strings = 0
else:
    self.approved_strings = F('approved_strings') + approved_strings_diff
    self.fuzzy_strings = F('fuzzy_strings') + fuzzy_strings_diff
    self.unreviewed_strings = F('unreviewed_strings') + unreviewed_strings_diff
```

Reply:
Agreed! :D